CHANGELOG.md

```diff
@@ -13,7 +13,7 @@
     - Interface ScmInlineAction removes 'commands: CommandRegistry'
     - Interface ScmInlineActions removes 'commands: CommandRegistry'
     - Interface ScmTreeWidget.Props removes 'commands: CommandRegistry'
-
+- [core] changed default icon theme from `none` to `theia-file-icons` [#11028](https://github.com/eclipse-theia/theia/pull/11028)
```

Comment:
minor: when you're ready to submit the pr make sure that the formatting is fine here:

- put the entry at line 11 since it comes first alphabetically
- make sure there is a newline before the `## v1.35.0 - 02/23/2023` entry
- use the proper link (of this pull-request not the new one) - need to create the pr first, then amend